pkg/registry/apis/provisioning/repository/github.go

```diff
@@ -57,18 +56,14 @@ func NewGitHub(
 	secrets secrets.Service,
 	webhookURL string,
 	cloneFn CloneFn,
-) (*githubRepository, error) {
-	owner, repo, err := parseOwnerRepo(config.Spec.GitHub.URL)
-	if err != nil {
-		return nil, err
-	}
+) *githubRepository {
```

Comment:
previously we failed completely when the config was bad -- that meant we could not get any details on why.